### DISCUSSION TITLE
Installing HTML documentation

### DISCUSSION THREAD

**Discussion by comburent** [jcorporation/myMPD#1128] [Ideas]
Feature request:
Having the HTML docs installed for a better self-contained/hosted app.

Suggestion:
- Generate and install the HTML documentation of <https://jcorporation.github.io/myMPD/> to a path specified using a new `-Dhtmldir=…` cmake option. (defaults to `/usr/share/doc/mympd/html/…` if not specified)
This step could be made optional through `MYMPD_HTMLDOC` for distributions not interested in it.

**Comment by jcorporation** [comment]
The HTML documentation is created by jekyll and the API documentation parts by doxygen and jsdoc. Therefor installing the html documentation locally requires many big dependencies.

**Comment by jcorporation** [comment]
Next version of myMPD will install as default the CHANGELOG.md, LICENSE.MD, README.md and SECURITY.md files. There is now also an option create and install the html documentation.

| OPTION | DEFAULT | DESCRIPTION |
| ------ | ------- | ----------- |
| MYMPD_DOC | ON | Installs documentation |
| MYMPD_DOC_HTML | OFF | Creates and installs the html documentation |
